android/src/main/java/com/idehub/GoogleAnalyticsBridge/GoogleAnalyticsBridge.java

```diff
@@ -251,6 +251,14 @@ public void trackSocialInteraction(String trackerId, String network, String acti
         }
     }
 
+	@ReactMethod
+	public void trackContentGroup(String trackerId, int contentGroupIndex, String category) {
+		Tracker tracker = getTracker(trackerId);
+		if (tracker != null) {
+			tracker.set(contentGroup(contentGroupIndex),category);	
```

Comment:
Like you said, documentation about this is really scarce. Here's the page I got most of the info from. You can see in the Android snippet it uses `contentGroup`. Not sure if this is global or not. https://support.google.com/analytics/answer/2853546?hl=en